specification/responses/4xx/message_batches/422_DuplicateBatchRequest.yaml

```diff
@@ -0,0 +1,21 @@
+description: |+
+  Request already received and it will be ignored.
+
+  ### Sandbox
+
+  It is possible to trigger this on the sandbox by using the `Prefer` header with a value of `code=422`.
```

Comment:
```suggestion
  It is possible to trigger this on the sandbox by using the `Prefer` header with a value of `code=422_message` or `code=422_batch`.
```

Reply:
Renamed to `code=422_batch` as this is what this specific error was referring to. I also update the message one to `code=422_message`.